config/environments/development.rb

```diff
@@ -12,6 +12,16 @@
   # Show full error reports and disable caching.
   config.consider_all_requests_local       = true
   config.action_controller.perform_caching = false
+  config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }
+  config.action_mailer.smtp_settings = {
+    address: "smtp.gmail.com",
+    port: 587,
+    domain: "gmail.com",
+    authentication: "plain",
+    user_name: "akshay.chhikara@vinsol.com",
```

Comment:
may u can use gem 'figaro' and add your secret configuration in a separate file and then we can add that configuration on heroku as well